config/docker/api-catalog-services-standalone.yml

```diff
@@ -6,8 +6,8 @@ logging:
 
 eureka:
     client:
-        initialInstanceInfoReplicationIntervalSeconds: 1
-        registryFetchIntervalSeconds: 1
+        instanceInfoReplicationIntervalSeconds: 30
```

Comment:
here also, the property is different

Reply:
The question is whether it was intentional before, default value for initialInstanceInfoReplicationIntervalSeconds=40, I would revert it to the default as well